deployment/requirements/dev.in

```diff
@@ -21,3 +21,4 @@ types-six
 types-tornado
 wheel
 debug
+lxml
```

Comment:
There is no need to duplicate this dependency between `dev.in` and `prod.in` (this is a production dependency). Inside `dev.in` we call `-r ./prod.in` (1st line of file) so that not only development dependencies but also the production dependencies are installed.